tests/DataStructures/QueueTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+use PHPUnit\Framework\TestCase;
```

Comment:
To follow PSR-12 guidelines, to make the linter happy - should have one blank line between header blocks.

```suggestion
<?php

use PHPUnit\Framework\TestCase;
```